### PR TITLE
Release prep for 0.8.0

### PR DESCRIPTION
**Pull request by esc**

### CODE REVIEW

**Comment by esc** [comment]
I modified the ``ANNOUNCE.rst`` fairly heavily to include the things I mentioned in https://github.com/Blosc/bcolz/issues/108. I'd appreciate if those of you who have some stakes in this (@FrancescElies, @CarstVaartjes, @mrocklin and @ehebert (or @twiecki)) to check if the mentions are correct, in your interest and if the wording is suitable. Thanks!

**Comment by esc** [comment]
Tests pass on Windows too.

File: ANNOUNCE.rst [hunk 1]
```diff
@@ -43,6 +49,25 @@ bcolz has minimal dependencies (NumPy), comes with an exhaustive test
...
+
+Other users of bcolz are the Blaze project (http://blaze.pydata.org/)
+and Quantopian (https://www.quantopian.com/) which you can read more
+about by pointing your browser at the links below.
+
+* Blaze links:
+
+  http://nbviewer.ipython.org/url/blaze.pydata.org/notebooks/timings-bcolz.ipynb
+  http://matthewrocklin.com/blog/work/2014/12/30/Towards-OOC-Frontend/
+  http://matthewrocklin.com/blog/work/2014/12/27/Towards-OOC/
```
Comment (mrocklin) [inline ANNOUNCE.rst]: Suggest the following links

```Python
Blaze Project: http://blaze.pydata.org/docs/dev/index.html
Notebooks showing Blaze + Pandas + BColz interaction: 
* http://nbviewer.ipython.org/url/blaze.pydata.org/notebooks/timings-csv.ipynb
* http://nbviewer.ipython.org/url/blaze.pydata.org/notebooks/timings-bcolz.ipynb
```

The OOC posts are pretty experimental and not necessarily blaze related.  I can point you to other blaze related blogposts if that's ideal.  I think that the links above are good though.

Comment (CarstVaartjes) [inline ANNOUNCE.rst]: Sounds great! If you want, you can add a link to bquery for people lazy to google ;) https://github.com/visualfabriq/bquery

**Comment by CarstVaartjes** [comment]
Sounds great! If you want, you can add a link to bquery for people lazy to google ;) https://github.com/visualfabriq/bquery

**Comment by esc** [comment]
Thanks for the feedback everyone I have updated the announcement as suggested and will give the Quantopian folks until tomorrow and then ship it!

**Comment by esc** [comment]
It's going down now!

**Comment by esc** [comment]
It went down, I probably shouldn't tell anyone, but the test suite segfaults my python debug build I made for debugging a differnt segfault... gdb claims it's in ``malloc.c`` ... I hope it is just a glitch of me incorrectly building the debug build.

**Pull request merged**